src/OrchardCore/OrchardCore.Infrastructure.Abstractions/Security/StandardPermissions.cs

```diff
@@ -6,4 +7,9 @@ public static class StandardPermissions
 {
     [Obsolete("This permission is deprecated and will be removed in future releases. Instead, consider adding users to the system administrator role.")]
     public static readonly Permission SiteOwner = new("SiteOwner", "Site Owners Permission", isSecurityCritical: true);
+
+    /// <summary>
+    /// This claim is assigned by the system during the login process if the user belongs to the Administrator role.
+    /// </summary>
+    public static readonly Claim SiteOwnerClaim = new("OrchardCorePermissions", "all-permissions");
```

Comment:
I think the type name should be more clear and maybe not related to our permissions only. In the future, we may want to use this claim for other checks, beside checking for the existing permissions.
Something like `IsSuperUser = true` or similar.

Reply:
I changed it to `Claim SiteOwner = new("SiteOwner", "true");` and moved it to a new `StandardClaims` class instead